xbmc/FileItem.cpp

```diff
@@ -2021,7 +2021,7 @@ void CFileItemList::Sort(SortDescription sortDescription)
   }
 
   // replace the current list with the re-ordered one
-  m_items.assign(sortedFileItems.begin(), sortedFileItems.end());
+  m_items = std::move(sortedFileItems);
```

Comment:
oh sorry, you're right about it. Assumed it was a CFileItemList as that's most commonly used in that part of code.